website/src/logged_in/logs/Logs.js

```diff
@@ -76,6 +87,106 @@ const columns = [
             }
         }
     },
+    {
+        field: 'daystoSetup',
+        headerName: 'Number of SetupDays',
+        description: 'Number of SetupDays',
+        sortable: true,
+        width: 200,
+        valueGetter: (params) => {
+            try {
+                return `${params.getValue('prediction_parameters')['setup_days']}`
+            } catch (error) {
+                return '';
+            }
+        }
+    },
+    {
+        field: 'daystoTeardown',
+        headerName: 'Number of TearDown Days',
+        description: 'Number of TearDown Days',
+        sortable: true,
+        width: 200,
+        valueGetter: (params) => {
+            try {
+                return `${params.getValue('prediction_parameters')['teardown_days']}`
+            } catch (error) {
+                return '';
+            }
+        }
+    },
+    {
+        field: 'isAudiovisual',
+        headerName: 'isAudioVisual',
+        description: 'isAudioVisual',
+        sortable: true,
+        width: 150,
+        valueGetter: (params) => {
+            try {
+                return `${params.getValue('prediction_parameters')['is_audio']}`
+            } catch (error) {
+                return '';
+            }
+        }
+    },
+    {
+        field: 'isTelecom',
+        headerName: 'isTelecom',
+        description: 'isTelecom',
+        sortable: true,
+        width: 150,
+        valueGetter: (params) => {
+            try {
+                return `${params.getValue('prediction_parameters')['is_audio']}`
+            } catch (error) {
+                return '';
+            }
+        }
+    },
+    {
+        field: 'spaceUsed',
+        headerName: 'Square Footage Utilized',
+        description: 'Square Footage Utilized',
+        sortable: true,
+        width: 225,
+        valueGetter: (params) => {
+            try {
+                return `${params.getValue('prediction_parameters')['sqft']}`
+            } catch (error) {
+                return '';
+            }
+        }
+    },
+    {
+        field: 'numAttend',
+        headerName: 'Forecast Attendence',
+        description: 'Forecast Attendence',
+        sortable: true,
+        width: 225,
+        valueGetter: (params) => {
+            try {
+                return `${params.getValue('prediction_parameters')['forecast_attendance']}`
+            } catch (error) {
+                return '';
+            }
+        }
+    },
+    {
+        field: 'totalDays',
+        headerName: 'Total Days (In-Out)',
```

Comment:
Renaming this to "Event Duration (Days)" would be easier for customers to understand.